app/domain/authentication/authn_jwt/validate_and_decode/fetch_issuer_value.rb

```diff
@@ -0,0 +1,149 @@
+require 'uri'
+
+module Authentication
+  module AuthnJwt
+    module ValidateAndDecode
+      # FetchIssuerValue command class is responsible to fetch the issuer secret value,
+      # in order to validate it later against the JWT token claim
+      FetchIssuerValue ||= CommandClass.new(
+        dependencies: {
+          resource_class: ::Resource,
+          fetch_secrets: ::Conjur::FetchRequiredSecrets.new,
```

Comment:
please change to `fetch_required_secrets`

Reply:
Done